### ISSUE TITLE
Bad command prompt on Windows 8.1

### ISSUE THREAD

**Issue by gerardog** [gerardog/gsudo#30]
On Windows 8.1 The elevated prompt shows:
```
C:\Users\Gerardo←[1;31;40m# ←[0;37;40m
```
instead of:
```
C:\Users\Gerardo# 
```
... which can be fixed with `gsudo config Prompt "$p# "`
but should work out of the box.

![image](https://user-images.githubusercontent.com/3901474/79631644-1d1c5c80-8131-11ea-9ce6-3afe1f0c5434.png)

**Comment by Krrishdhaneja** [comment]
I think it can be a git(or any other software error) error or something like that @gerardog cuz when I installed full git sdk I got the same and when I uninstalled it and installed the normal git then it was the same prompt!

**Comment by gerardog** [comment]
HI @Krrishdhaneja.
gsudo uses ANSI codes to show the red `#`. For this to work, the user terminal must support ANSI colors. If it doesn't, it wont be able to interpret `←[1;31;40m` as a color change command and will display it as plain text.
Windows 8.1 (well, actually any windows version before Win10 Anniversary Update) built-in console (`conhost.exe`) does not support ANSI colors out of the box and require some kind of workaround such as the ANSICON utility. 

Your scenario looks different. I assume you are using Windows 10 Anniversary build or later, so your CONHOST works fine. And you installed some git version that included another terminal console which does not support ANSI and fails.

If that is the case, you either use a console that supports ANSI or run `gsudo config Prompt "$p# "` to disable the red color.

This issue was about improving support on older windows versions, prior to `Win 10 Anniversary Update`. I may be wrong, but I think that detecting if the current console supports ANSI is a hard problem. So the easiest way would be to detect if Windows version is prior to Anniversary, then just change the default prompt to the non colored #.

Then the problem is how to detect the exact Windows version. It should be trivial but found there is a grotesque amount of compatibility layers that won't allow you to detect it easily and really fast, and not including a manifest that would fatten `gsudo.exe`.

**Comment by gerardog** [comment]
``` 
        var osVersionInfo = new OSVERSIONINFOEX { OSVersionInfoSize = Marshal.SizeOf(typeof(OSVERSIONINFOEX)) };
        if (!RtlGetVersion(ref osVersionInfo))
        {
            Console.WriteLine($"{osVersionInfo.MajorVersion}.{osVersionInfo.MinorVersion}.{osVersionInfo.BuildNumber}");
        }

        [System.Security.SecurityCritical]
        [DllImport("ntdll.dll", SetLastError = true, CharSet = CharSet.Unicode)]
        internal static extern bool RtlGetVersion(ref OSVERSIONINFOEX versionInfo);

        [StructLayout(LayoutKind.Sequential, CharSet=CharSet.Unicode)]
        internal struct OSVERSIONINFOEX
        {
            // The OSVersionInfoSize field must be set to Marshal.SizeOf(typeof(OSVERSIONINFOEX))
            internal int OSVersionInfoSize;
            internal int MajorVersion;
            internal int MinorVersion;
            internal int BuildNumber;
            internal int PlatformId;
            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 128)]
            internal string CSDVersion;
            internal ushort ServicePackMajor;
            internal ushort ServicePackMinor;
            internal short SuiteMask;
            internal byte ProductType;
            internal byte Reserved;
        }
```

**Comment by gerardog** [comment]
Fix released in [v1.0.0](https://github.com/gerardog/gsudo/releases/tag/v1.0.0) (PR #94)
